be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -286,15 +286,10 @@ Status SinkBuffer::_try_to_send_rpc(const TUniqueId& instance_id, const std::fun
 
         TransmitChunkInfo& request = buffer.front();
         bool need_wait = false;
-        DeferOp pop_defer([&need_wait, &buffer, mem_tracker = _mem_tracker]() {
+        DeferOp pop_defer([&need_wait, &buffer]() {
             if (need_wait) {
                 return;
             }
-
-            // The request memory is acquired by ExchangeSinkOperator,
-            // so use the instance_mem_tracker passed from ExchangeSinkOperator to release memory.
-            // This must be invoked before decrease_defer desctructed to avoid sink_buffer and fragment_ctx released.
-            SCOPED_THREAD_LOCAL_MEM_TRACKER_SETTER(mem_tracker);
             buffer.pop();
```

Comment:
This may still need. Because the request memory is allocated by driver threads (`ExchangeSinkOperator::push_chunk`), but `_try_to_send_rpc` may be invoked in bthreads.

Reply:
Got it.